src/styles/clock.scss

```diff
@@ -0,0 +1,64 @@
+.container__clock {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%,-50%);
+  width: 500px;
+  height: 500px;
+  background: url(../imageclock.jpg);
+  background-size: cover;
+  border: 8px solid black;
+  border-radius: 50%;
+  transition-delay: 2s;
+
+  &--arrow::before {
+    content: "";
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+    width: 35%;
+    height: 6px;
+    background: blue;
+    border-radius: 3px;
+    animation: animate 4s linear infinite;
+    transform-origin: left;
+  }
+
+  &--arrow::after {
+    content: "";
+    position: absolute;
```

Comment:
you duplicate a lot of properties
you can create a separate div for each `arrow` (`arrow` as BEM-block) and write shared properties to `arrow` selector and different properties in modificators (as `arrow-minute` etc.)